homeassistant/components/roku/config_flow.py

```diff
@@ -53,20 +58,38 @@ def __init__(self) -> None:
         self.discovery_info = {}
 
     @callback
-    def _show_form(self, errors: dict[str, Any] | None = None) -> ConfigFlowResult:
+    def _show_form(
+        self,
+        errors: dict[str, Any] | None = None,
+        user_input: dict[str, Any] | None = None,
```

Comment:
```suggestion
        user_input: dict[str, Any] | None
        errors: dict[str, Any] | None = None,
```
then you don't need to pass `None` explicity below, and user input seems like it's required anyway to me